apps/website/src/content/docs/table.mdx

```diff
@@ -6,16 +6,466 @@ thumbnail: Table
+
+<LiveExample src='Table.selectMulti.jsx'>
+  <AllExamples.TableSelectMultiExample client:load />
+</LiveExample>
+
+### Single-row selection
+
+In single row selection mode (`selectionMode="single"`), users can select each row by clicking on the desired row.
+
+**Note**: For accessibility reasons, in single-selection Tables, it is highly recommended to include a button in the primary cell of each row. [More info](https://github.com/iTwin/iTwinUI/issues/2130#issuecomment-2209287280).
+
+```jsx
+const columns = React.useMemo(
+  () =>
+    [
+      /* … */
+      {
+        /* … */
+        Cell: ({ value }) => {
+          return (
+            <Anchor
+              as="button"
+              onClick={() => console.log(`Selected ${value}`)}
+            >
+              {value}
+            </Anchor>
+          );
+        },
+      },
+    ] satisfies Column[],
+  []
+);
+```
+
+<LiveExample src='Table.selectSingle.jsx'>
+  <AllExamples.TableSelectSingleExample client:load />
+</LiveExample>
+
+### Prevent accidental row selection
+
+By default, `selectRowOnClick` is `true`. This is ideal for most cases. However, this can be disabled in use-cases where the loss of the selection state due to an accidental row click is too disruptive.
+
+```jsx {3}
+<Table
+  /* … */
+  selectRowOnClick={false}
+/>
+```
+
+To prevent the selection of the row when just hoping to select the cell content, we add a wrapper to increase the hit size of the cell content. However, this does not happen if a `cellRenderer` is passed with custom children.
+
+```jsx
+const columns = React.useMemo(
+  () => [
+    // hit-size increasing wrapper added since no cellRenderer.
+    {
+      /* … */
+    },
+
+    // hit-size increasing wrapper added since
+    // <DefaultCell> children is not overridden.
+    {
+      /* … */
+      cellRenderer: (props) => <DefaultCell {...props} />,
+    },
+
+    // hit-size increasing wrapper NOT added since
+    // <DefaultCell> children is overridden.
+    {
+      /* … */
+      cellRenderer: (props) => (
+        <DefaultCell {...props}>
+          <span>{props.value}</span>
+        </DefaultCell>
+      ),
+    },
+  ],
+  [],
+);
+```
+
+To prevent the selection of the row when clicking on custom content in `Cell` or `cellRenderer`, you may need to stop click event propagations.
+
+```jsx {10, 14}
+const columns = React.useMemo(
+  () => [
+    /* … */
+    {
+      ...ActionColumn(),
+      Cell: () => {
+        return (
+          <DropdownMenu
+            menuItems={menuItems}
+            onClick={(e) => e.stopPropagation()} // 👈
+          >
+            <IconButton
+              styleType='borderless'
+              onClick={(e) => e.stopPropagation()} // 👈
+              aria-label='More options'
+            >
+              <SvgMore />
+            </IconButton>
+          </DropdownMenu>
+        );
+      },
+    },
+  ],
+  [],
+);
+```
+
+## Sorting
```

Comment:
As the "Selection" section has become long, I'm thinking "Sorting" should come before "Selection" for better flow.